integration/src/test/scala/ammonite/integration/BasicTests.scala

```diff
@@ -25,6 +25,27 @@ object BasicTests extends TestSuite{
       assert(evaled.out.trim == "Hello World")
     }
 
+    //make sure scripts with symbols in path names work fine
+    'scriptWithSymbols {
+      if (!Util.windowsPlatform){
+        val scriptAddr =
+          cwd/'integration/'target/'test/'resources/'ammonite/'integration/'basic/"script%#.@*+叉燒.sc"
+        rm(scriptAddr)
```

Comment:
I have changed scriptAddr from src to target
I think this one should be better.